samples/electron-app/src/main/main.ts

```diff
@@ -0,0 +1,69 @@
+import { app, BrowserWindow } from 'electron';
+import * as path from 'path';
+import { ElectronAuthenticator } from '@microsoft/mgt-electron-provider/dist/Authenticator/ElectronAuthenticator';
```

Comment:
We can simplify this to `import { ElectronAuthenticator } from '@microsoft/mgt-electron-provider/dist/Authenticator` by adding an index.ts file in the src/Authenticator folder that reexports the `ElectronAuthenticator`

Same for `ElectronProvider`